beacon-chain/powchain/service.go

```diff
@@ -189,9 +188,9 @@ func (w *Web3Service) ChainStartFeed() *event.Feed {
 	return w.chainStartFeed
 }
 
-// ChainStartDeposits returns a slice of validator deposits processed
+// ChainStartDeposits returns a slice of validator deposit data processed
 // by the deposit contract and cached in the powchain service.
-func (w *Web3Service) ChainStartDeposits() []*pb.Deposit {
+func (w *Web3Service) ChainStartDeposits() [][]byte {
```

Comment:
We need this in raw bytes to easily reconstruct the deposit trie when needed after the voting period is over when fetching pending deposits